README.md

```diff
@@ -10,7 +10,7 @@ Check out this [deployment tutorial](https://code.visualstudio.com/tutorials/app
 
 * Browse sites across all of your Azure subscriptions
 * Browse to the Azure Portal for advanced tasks, such as scaling
-* Create new web apps/deployment slots (Linux with Node.js only)
+* Create new web apps/deployment slots (Linux and Windows apps)
```

Comment:
I'd just remove the section in the parentheses at this point since we support everything app service supports